File: app/src/main/java/jp/co/pannacotta/norokoro/SplashActivity.java
package jp.co.pannacotta.norokoro;

import android.content.Intent;
import android.os.Bundle;
import android.os.Handler;
import android.os.Looper;
import android.support.annotation.Nullable;
import android.support.v7.app.AppCompatActivity;

public class SplashActivity extends AppCompatActivity {
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_splash);

        new Handler(Looper.getMainLooper()).postDelayed(new Runnable() {
            @Override
            public void run() {
                if (!isFinishing()) {
                    // isFinishingでチェックしないとバックキーを押したとき登録画面にいってしまう
                    Intent intent = new Intent(SplashActivity.this, TourokuFragment.class);
                    startActivity(intent);
                    finish();
                }
            }
        }, 2000);
    }
}

File: app/src/main/java/jp/co/pannacotta/norokoro/SplashActivity.kt
package jp.co.pannacotta.norokoro

import android.content.Intent
import android.os.Bundle
import android.os.Handler
import android.os.Looper
import android.support.v7.app.AppCompatActivity

class SplashActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_splash)
        Handler(Looper.getMainLooper()).postDelayed({
            if (!isFinishing) {
                // isFinishingでチェックしないとバックキーを押したとき登録画面にいってしまう
                val intent = Intent(this@SplashActivity, TourokuActivity::class.java)
                startActivity(intent)
                finish()
            }
        }, 2000)
    }
}